packages/docusaurus-theme-common/src/utils/ThemeClassnames.ts

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
camel case the title too

Reply:
Does not look like resolved?

![image](https://user-images.githubusercontent.com/749374/113692323-b9869880-96cd-11eb-88f5-be10ebe16f4e.png)